cmake error trying to build coarray_navier_stokes

The change in commit 3d06a26 wasn't enough to disable the coarray_navier_stokes test. This is the error on Centos 7.3:
````
CMake Error at src/tests/integration/pde_solvers/navier-stokes/CMakeLists.txt:11 (add_executable):
  Cannot find source file:

    EXCLUDE_FROM_ALL

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


CMake Error: CMake can not determine linker language for target: coarray_navier_stokes
CMake Error: Cannot determine link language for target "coarray_navier_stokes".
-- Generating done
````

--- a/.pullapprove.yml
+++ b/.pullapprove.yml
@@ -11,19 +11,20 @@ groups:
         - master
         - devel
         - caffeinate-opencoarrays
-      files:
-        exclude:
-          - "*.md"
-    users:
-      - afanfa
-      - rouson
-      - zbeekman
+    teams:
+      - team-opencoarrays
         # https://github.com/orgs/sourceryinstitute/teams/team-opencoarrays
   OpenCoarrays:
     # Whenever someone has a PR, on any branch require at least 1 
     # collaborator to explicitly review it
     required: 1
     users: all
+    conditions:
+      branches:
+        exclude:
+          - master
+          - devel
+          - caffeinate-opencoarrays
   ConfigManagers:
     required: 1
     conditions:

--- a/src/tests/integration/pde_solvers/navier-stokes/CMakeLists.txt
+++ b/src/tests/integration/pde_solvers/navier-stokes/CMakeLists.txt
@@ -9,9 +9,9 @@ if ( ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "x86_64") AND ("${CMAKE_SYSTEM_NAME}" 
       set(fft_library ${CMAKE_CURRENT_SOURCE_DIR}/libfft_avx.a )
     endif()
     add_executable( coarray_navier_stokes
+      EXCLUDE_FROM_ALL
       coarray-shear_coll.F90
       ${walltime_o}
-      EXCLUDE_FROM_ALL
     )
     target_link_libraries(coarray_navier_stokes OpenCoarrays ${fft_library})
   endif()

